src/test/kotlin/calculator/CalculatorTest.kt

```diff
@@ -29,4 +29,13 @@ class CalculatorTest {
 
         assertThat(calculator.calculate("2 + 9 - 1 * 10 / 5")).isEqualTo(20)
     }
+
+    @Test
+    fun `number list와 symbol list를 순서대로 조합해서 계산하고 결과를 리턴한다`() {
```

Comment:
```suggestion
    fun `피연산자와 연산자를 조합해서 계산한 값을 반환한다`() {
```
테스트 함수명을 좀 더 이해하기 쉽게 작성해보면 어떨까요?